tests/test_asyncio.py

```diff
@@ -1,88 +0,0 @@
-from graphql.execution.executors.asyncio import AsyncioExecutor
-from graphql.type.definition import GraphQLField, GraphQLNonNull, GraphQLObjectType
-from graphql.type.scalars import GraphQLString
-from graphql.type.schema import GraphQLSchema
-from promise import Promise
-
-import asyncio
-from graphql_server import RequestParams, run_http_query
-
-from .utils import as_dicts
-
-
-def resolve_error_sync(_obj, _info):
-    raise ValueError("error sync")
-
-
-async def resolve_error_async(_obj, _info):
-    await asyncio.sleep(0.001)
-    raise ValueError("error async")
-
-
-def resolve_field_sync(_obj, _info):
-    return "sync"
-
-
-async def resolve_field_async(_obj, info):
-    await asyncio.sleep(0.001)
-    return "async"
-
-
-NonNullString = GraphQLNonNull(GraphQLString)
-
-QueryRootType = GraphQLObjectType(
-    name="QueryRoot",
-    fields={
-        "errorSync": GraphQLField(NonNullString, resolver=resolve_error_sync),
-        "errorAsync": GraphQLField(NonNullString, resolver=resolve_error_async),
-        "fieldSync": GraphQLField(NonNullString, resolver=resolve_field_sync),
-        "fieldAsync": GraphQLField(NonNullString, resolver=resolve_field_async),
-    },
-)
-
-schema = GraphQLSchema(QueryRootType)
-
-
-def test_get_responses_using_asyncio_executor():
```

Comment:
Did these tests get moved anywhere?

Reply:
Nop. Probably that was a mistake, I don't remember the reasoning behind this removal.

Reply:
@jkimbo @Cito I remember why this file was removed: core package has no asyncio executor as far as I know as I asked about it on the Slack channel.